podcasts/CarPlaySceneDelegate+Interaction.swift

```diff
@@ -51,17 +51,20 @@ extension CarPlaySceneDelegate {
 
         // Anything else, load the episode and start playing it
         PlaybackManager.shared.load(episode: episode, autoPlay: true, overrideUpNext: false)
+
+        // Store the playlist
+        AutoplayHelper.shared.playedFrom(playlist: playlist)
```

Comment:
Ideally, this will be in a single place — that place being `load`. However, `load` is used in multiple places so I think we can live with this for now.